dojo_theme/static/js/dojo/popup.js

```diff
@@ -0,0 +1,119 @@
+
+const BELT_ORDER = ["white", "orange", "yellow", "green", "blue", "black"];
+const BELT_MESSAGES = {
+    white: "<b>You've earned your white belt!</b><br>The journey begins!",
+    orange: "<b>Orange belt achieved!</b><br>Vulnerability Seeker - You know where to strike first.",
+    yellow: "<b>Yellow belt unlocked!</b><br>Buffer Breaker - Exploiting memory, one byte at a time.",
+    green: "<b>Green belt obtained!</b><br>Precision Hacker - No gap too small, no exploit too complex.",
+    blue: "<b>Blue belt conquered!</b><br>Master of the System - You&#39;ve conquered the complexity others fear.",
+    black: "<b>Black belt Mentor!</b><br> Architect of Understanding - You forge minds as sharp as your exploits."
+};
+
+document.addEventListener("DOMContentLoaded", function () {
+    if (!init?.userId) return;
+    
+    checkUserAwards()
+        .then(handleAwardPopup)
+        .catch(error => console.error("Award check failed:", error));
+});
+
+function checkUserAwards() {
+    const endpoint = `${CTFd.config.urlRoot}/api/v1/users/${init.userId}/awards`;
+    return fetch(endpoint, {
+        method: "GET",
+        credentials: "same-origin",
+        headers: {"Accept": "application/json"}
+    }).then(res => res.json());
+}
+
+function handleAwardPopup(response) {
+    if (!response?.success || !response.data?.length) return;
+
+    const awards = response.data;
+
+    const beltAwards = awards.filter(a => BELT_ORDER.includes(a.name)).sort(dateSort);
+    const otherAwards = awards.filter(a => !BELT_ORDER.includes(a.name)).sort(dateSort);
+    const latestAward = beltAwards.pop() || otherAwards.pop();
+    
+    const lastSeen = localStorage.getItem("lastAwardDate") || "1970-01-01";
+    if (new Date(latestAward.date) <= new Date(lastSeen)) return;
+    
+
+    localStorage.setItem("lastAwardDate", latestAward.date);
+    showAwardPopup(latestAward);
+}
+
+function dateSort(a, b) {
+    return new Date(a.date) - new Date(b.date);
+}
+
+function showAwardPopup(award) {
+    const isBelt = BELT_ORDER.includes(award.name);
+
+    const imageContent = isBelt ? 
+    `<img src = "https://pwn.college/belt/${award.name}.svg" 
```

Comment:
We don't want to reference "https://pwn.college" in source since the dojo is open source and should work on other domains.

I assume a relative url of just "/belt/..." should work.